pkg/dss/scd/cockroach/store.go

```diff
@@ -0,0 +1,124 @@
+package cockroach
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"fmt"
+
+	"github.com/dpjacques/clockwork"
+	"go.uber.org/zap"
+)
+
+var (
+	// DefaultClock is what is used as the Store's clock, returned from Dial.
+	DefaultClock = clockwork.NewRealClock()
+)
+
+// Store is an implementation of dss.Store using
+// Cockroach DB as its backend store.
+type Store struct {
+	*sql.DB
+	logger *zap.Logger
+	clock  clockwork.Clock
+}
+
+// Dial returns a Store instance connected to a cockroach instance available at
+// "uri".
+// https://www.cockroachlabs.com/docs/stable/connection-parameters.html
+func Dial(uri string, logger *zap.Logger) (*Store, error) {
```

Comment:
It might be nice to allow the caller to use one connection for both scd and rid. We could do something like have a Store.WithDB() or something

Reply:
Added #261